Modules/_datetimemodule.c

```diff
@@ -40,6 +40,27 @@
 
 #define PyTimezone_Check(op) PyObject_TypeCheck(op, &PyDateTime_TimeZoneType)
 
+typedef struct {
+    /* Conversion factors. */
+    PyObject *us_per_ms;       // 1_000
+    PyObject *us_per_second;   // 1_000_000
+    PyObject *us_per_minute;   // 1e6 * 60 as Python int
+    PyObject *us_per_hour;     // 1e6 * 3600 as Python int
+    PyObject *us_per_day;      // 1e6 * 3600 * 24 as Python int
+    PyObject *us_per_week;     // 1e6 * 3600 * 24 * 7 as Python int
+    PyObject *seconds_per_day; // 3600 * 24 as Python int
+
+    /* The interned UTC timezone instance */
+    PyObject *utc;
+
+    /* The interned Epoch datetime instance */
```

Comment:
I don't think that "interned" is correct or important here. But it would be useful to mention its value:

```
datetime.datetime(1970, 1, 1, tzinfo=datetime.timezone.utc)
```

You may mention that it's also the Unix epoch ;-) https://en.wikipedia.org/wiki/Unix_time

Reply:
I deliberately did not alter the comments. I'm not sure it matters to remove "interned" or not; as you say, it is not important. Clarifying that "epoch" is the "Unix epoch" is ok, although I'd expect most C developers to be familiar with that term.

Reply:
There are multiple "epochs" :-) https://en.wikipedia.org/wiki/Epoch_(computing)#Notable_epoch_dates_in_computing